docs/configuration/config-file-reference.md

```diff
@@ -1028,315 +1028,319 @@ The `table_manager_config` configures the Cortex table-manager.
 # CLI flag: -table-manager.retention-period
 [retention_period: <duration> | default = 0s]
 
-# How frequently to poll DynamoDB to learn our capacity.
-# CLI flag: -dynamodb.poll-interval
-[dynamodb_poll_interval: <duration> | default = 2m0s]
+# How frequently to poll backend to learn our capacity.
+# CLI flag: -table-manager.poll-interval
+[poll_interval: <duration> | default = 2m0s]
```

Comment:
Remember to mention this YAML config rename (`dynamodb_poll_interval` to `poll_interval`) in the changelog